civictechprojects/static/css/partials/_IconLinkDisplay.scss

```diff
@@ -46,7 +46,7 @@
     text-overflow: ellipsis;
   }
 }
-.IconLink-right:hover, .IconLink-right:active {
+.IconLink-root:hover p, .IconLink-root:active p {
```

Comment:
Making it so the hover state applies no matter where your mouse cursor is, previously it wouldn't work properly if you hovered over the icon on the left rather than the text on the right. Gives better hinting that the whole box is clickable this way.